account_invoice_report_by_partner/tests/test_account_invoice_report_by_partner.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import openerp.tests.common as common
+from datetime import datetime
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+def create_simple_invoice(self):
```

Comment:
I saw this way to do in other OCA project, but in this case will be better in class, I'm fixing it.